File: Android/app/src/main/java/com/felipearpa/tyche/poolhome/drawer/PoolHomeDrawerUiState.kt
package com.felipearpa.tyche.poolhome.drawer

import com.felipearpa.tyche.pool.PoolGamblerScoreModel
import com.felipearpa.ui.state.LoadState

data class PoolHomeDrawerUiState(
    val accountId: String,
    val email: String,
    val username: String,
    val poolGamblerScoreState: LoadState<PoolGamblerScoreModel>,
    val isOwner: Boolean,
    val gamblerCount: Int?,
    val isDeleting: Boolean,
)

File: iOS/Tyche/Tyche/UI/PoolHome/PoolHomeDrawerUiState.swift
import ViewingState
import Pool

struct PoolHomeDrawerUiState {
    let accountId: String
    let email: String
    let username: String
    let poolGamblerScoreState: LoadState<PoolGamblerScoreModel>
    let isOwner: Bool
    let gamblerCount: Int?
    let isDeleting: Bool
}
